roles/pmm/tasks/alerting-alerts.yaml

```diff
@@ -11,7 +11,7 @@
 - name: Check Template
   ansible.builtin.debug:
     msg: "Template name '{{ prefix_template_name }}' does not exist."
-  when: prefix_template_name not in pmm_existing_alert_templates
+  when: prefix_template_name not in pmm_updated_alert_templates
```

Comment:
@theTibi it appears that there is something slightly odd here. 
This set of tasks is working in a [task that loops](https://github.com/cezmunsta-org/gascan-ansible/blob/e06b686ae90c58095fb1e42477aaa097e7d56bbc/roles/pmm/tasks/grafana-alerting.yaml#L143-L146) and you are setting vars at that point. So, why reference the facts set outside of this loop, is it necessary? Can't you set those as vars outside and avoid quite a bit of code in these tasks?

Reply:
@cezmunsta If I understand right your comment, those vars should be inside the block because they are different for every alert like ```pmm_param_float: "{{ pmm_alert_list | json_query(\"[?name==`\"+ pmm_alert_item.name +\"`].threshold\") }}"``` so this is based on the `pmm_alert_item.name` and that will be different for every alert in the loop

Reply:
@theTibi what about this? https://github.com/cezmunsta-org/gascan-ansible/pull/13/files#diff-5c2b1d975c1f80e9a71539272084067ea7d6f2e1793b8d31ba0a3dad507eafa0R23 etc

Reply:
This one is also different for every `prefix_template_name` , basically I am selecting all the parameters for that specific template here and those has to be refreshed for every template. But if you have a better idea, let me know.

Reply:
Sure, I was pointing out that there are things defined inside that can be defined outside. 
`prefix_template_name` is determinable outside instead of as a task, right?